docs/research/tpm2-triage.md

```diff
@@ -0,0 +1,123 @@
+Report of TPM2 support on Voyage Linux 0.10.0
+=============================================
+
+## Conclusion
+
+Works with `tpm_tis` module bypassing the BIOS in TPM setup.
+
+## Actual steps taken
+
+HW used: APU2 with TPM LPC addon
+
+1. Install 4.13.0-rc6 kernel
```

Comment:
@mek-x I also believe some users may need info about kernel installation:

```
make -j$(nproc) deb-pkg                             
cd ..                                                                                               
scp *.deb root@192.168.0.104:/root  
```

on target:

```
cd /root
dpkg -i *.deb
````

I saw some problems during package installation - not sure who's fault it is:

```
root@voyage:~# dpkg -i *.deb                               
Selecting previously unselected package linux-firmware-image-4.13.0-rc6.                                               
(Reading database ... 18058 files and directories currently installed.)                                                
Preparing to unpack linux-firmware-image-4.13.0-rc6_4.13.0-rc6-2_amd64.deb ...                                         
Unpacking linux-firmware-image-4.13.0-rc6 (4.13.0-rc6-2) ...                                                           
Selecting previously unselected package linux-headers-4.13.0-rc6.                                                      
Preparing to unpack linux-headers-4.13.0-rc6_4.13.0-rc6-2_amd64.deb ...                                                
Unpacking linux-headers-4.13.0-rc6 (4.13.0-rc6-2) ...      
dpkg-deb (subprocess): decompressing archive member: lzma error: compressed data is corrupt                            
dpkg-deb: error: subprocess <decompress> returned error exit status 2                                                  
dpkg: error processing archive linux-headers-4.13.0-rc6_4.13.0-rc6-2_amd64.deb (--install):                            
 subprocess dpkg-deb --fsys-tarfile returned error exit status 2                                                       
^Bn^HSelecting previously unselected package linux-image-4.13.0-rc6.                                                   
Preparing to unpack linux-image-4.13.0-rc6_4.13.0-rc6-2_amd64.deb ...                                                  
Unpacking linux-image-4.13.0-rc6 (4.13.0-rc6-2) ...        
Preparing to unpack linux-libc-dev_4.13.0-rc6-2_amd64.deb ...
Unpacking linux-libc-dev (4.13.0-rc6-2) over (3.16.7-ckt25-1) ...
[  683.321422] traps: dpkg-deb[2581] trap invalid opcode ip:7f6713b49229 sp:7ffea98b14b0 error:0 in liblzma.so.5.0.0[7f6713b37000+22000]
dpkg-deb: error: subprocess <decompress> was killed by signal (Illegal instruction)
dpkg: error processing archive linux-libc-dev_4.13.0-rc6-2_amd64.deb (--install):
 cannot copy extracted data for './usr/include/linux/uhid.h' to '/usr/include/linux/uhid.h.dpkg-new': unexpected end of file or stream
Setting up linux-firmware-image-4.13.0-rc6 (4.13.0-rc6-2) ...
Setting up linux-image-4.13.0-rc6 (4.13.0-rc6-2) ...
update-initramfs: Generating /boot/initrd.img-4.13.0-rc6
live-boot: core filesystems devices utils udev wget blockdev.
Booting 4.13.0-rc6, old is 3.16.7-ckt9-voyage
Warning: Not updating LILO; /etc/lilo.conf not found!
Searching for GRUB installation directory ... found: /boot/grub
Probing devices to guess BIOS drives. This may take a long time.
[  726.591659] device-mapper: ioctl: 4.27.0-ioctl (2013-10-30) initialised: dm-devel@redhat.com
Searching for default file ... Generating /boot/grub/default file and setting the default boot entry to 0
Searching for GRUB installation directory ... found: /boot/grub
Testing for an existing GRUB menu.lst file ... found: /boot/grub/menu.lst
Searching for splash image ... none found, skipping ...
Found kernel: /boot/vmlinuz-4.13.0-rc6
Found kernel: /boot/vmlinuz-3.16.7-ckt9-voyage
Updating /boot/grub/menu.lst ... done

Errors were encountered while processing:
 linux-headers-4.13.0-rc6_4.13.0-rc6-2_amd64.deb
 linux-libc-dev_4.13.0-rc6-2_amd64.deb
root@voyage:~#  
```

That's probably my US, that's why we need working NFS for development.